src/app/api/utils/auth.ts

```diff
@@ -172,10 +168,10 @@ export const authOptions: AuthOptions = {
   },
   events: {
     signIn(message) {
-      log.debug("fxa-confirmed-profile-data", message.user);
+      logger.debug("fxa-confirmed-profile-data", message.user);
```

Comment:
PII check?

Reply:
oh since it's debug, prob won't log to gcp

Reply:
Thanks... I think we're capturing debug, I'll make it log something here that we can use for diagnostics but not the whole user object.